src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -832,10 +832,13 @@ public void changed(ImportTask task)  {
         changed(task.getContext());
     }
 
-    public Long runAsync(final ImportContext context, final ImportFilter filter) {
+    public Long runAsync(final ImportContext context, final ImportFilter filter, final boolean init) {
```

Comment:
In the interests of preserving API consistency, might it be better to overload this method with:
```
public Long runAsync(final ImportContext context, final ImportFilter filter) {
    runAsync(context, filter, false);
}
```